src/store/createStoreObservable.ts

```diff
@@ -9,18 +10,24 @@ global.Rx = { config: { Promise } };
  * only want to know about the end result of an operation, and don't want to deal with
  * any recoverable failures.
  */
-export interface StoreObservable<T, U> extends Observable<U> {
-	then<V>(onFulfilled?: ((value?: T[]) => (V | Promise<V> | null | undefined)) | null | undefined, onRejected?: (reason?: Error) => void): Promise<V>;
-}
+export type StoreObservable<T, U> = Subscribable<U> & Promise<T[]>
 
 export default function createStoreObservable<T, U>(observable: Observable<U>, transform: (data: U) => T[]): StoreObservable<T, U> {
-	const storeObservable = observable as StoreObservable<T, U>;
-	storeObservable.then = function<V>(onFulfilled?: ((value?: T[]) => (V | Promise<V> | null | undefined)) | null | undefined, onRejected?: (reason?: Error) => void): Promise<V> {
+	const storeObservable: any = observable;
```

Comment:
`observable`  can no longer be cast to the `StoreObservable` type because the types of `catch` and `finally` are not compatible.